include/SFML/Window/Event.hpp

```diff
@@ -100,7 +100,7 @@ class Event
     };
 
     ////////////////////////////////////////////////////////////
-    /// \brief Mouse wheel events parameters (MouseWheelMoved)
+    /// \brief Mouse wheel events parameters (MouseWheelMoved) (deprecated)
```

Comment:
How about `\deprecated This event is deprecated and potentially inaccurate. Use ... instead.`? Just saying it is deprecated is kinda pointless as a description.